tools/check_format.py

```diff
@@ -428,19 +452,33 @@ def checkErrorMessages(error_messages):
     return True
   return False
 
+
 if __name__ == "__main__":
   parser = argparse.ArgumentParser(description='Check or fix file format.')
-  parser.add_argument('operation_type', type=str, choices=['check', 'fix'],
-                      help="specify if the run should 'check' or 'fix' format.")
-  parser.add_argument('target_path', type=str, nargs="?", default=".",
-                      help="specify the root directory for the script to recurse over. Default '.'.")
-  parser.add_argument('--add-excluded-prefixes', type=str, nargs="+",
-                      help="exclude additional prefixes.")
-  parser.add_argument('-j', '--num-workers', type=int, default=multiprocessing.cpu_count(),
-                      help="number of worker processes to use; defaults to one per core.")
+  parser.add_argument(
```

Comment:
ping me when we've got these files i 100-char mode. I'm wondering if you may need to reset the Python files to their original state; it might be that the formatter will not be able to reverse its original line-length reduction, so you may need to use git magic for that.

Reply:
@jmarantz I believe this is intentional as the line without wrapping would be 112 characters. It appears the tool is wrapping all arguments if they can't fit into a single line.